doc/manual.rst

```diff
@@ -5865,6 +5865,26 @@ type `system.ForLoopStmt` can rewrite the entirety of a `for` loop:
   for a, b in enumerate(10, [1, 2, 3, 5]):
     echo a, " ", b
 
+Another example:
+
+.. code-block:: nim
```

Comment:
Actually, I think it would be preferable to switch the orders of the examples - have this one (which is much simpler) come first, and follow with the more complex one.